src/lib/logger.ts

```diff
@@ -1,20 +1,37 @@
 /* eslint-disable @typescript-eslint/no-explicit-any */
 
-/* Logger interface with a subset of methods from winston.js */
+// From https://github.com/winstonjs/winston/blob/v3.3.3/index.d.ts#L53
+type LogCallback = (
+  error?: any,
+  level?: string,
+  message?: string,
+  meta?: any
+) => void;
+
+// From https://github.com/winstonjs/winston/blob/v3.3.3/index.d.ts#L69-L75
+export interface LeveledLogMethod {
```

Comment:
I hesitate to add all these methods we don't know how to use.
Using this interface is fine, but I would trim it down to one or two methods and you can document how to use them